econml/solutions/causal_analysis/_causal_analysis.py

```diff
@@ -1065,18 +1100,23 @@ def _policy_tree_output(self, Xtest, feature_index, *, treatment_cost=0,
 
         Returns
         -------
-        tree : tuple of string, list of int
-            The policy tree represented as a graphviz string and the recommended treatment for each row
+        tree : tuple of string, float, float
+            The policy tree represented as a graphviz string,
+            the value of applying the recommended policy (over never treating),
+            the value of always treating (over never treating)
         """
 
-        intrp, feature_names, treatment_names, treat = self._tree(True, Xtest, feature_index,
-                                                                  treatment_cost=treatment_cost,
-                                                                  max_depth=max_depth,
-                                                                  min_samples_leaf=min_samples_leaf,
-                                                                  min_impurity_decrease=min_value_increase,
-                                                                  alpha=alpha)
-        return intrp.export_graphviz(feature_names=feature_names,
-                                     treatment_names=treatment_names), treat.tolist()
+        (intrp, feature_names, treatment_names,
+            (policy_val, always_trt)) = self._tree(True, Xtest, feature_index,
+                                                   treatment_cost=treatment_cost,
+                                                   max_depth=max_depth,
+                                                   min_samples_leaf=min_samples_leaf,
+                                                   min_impurity_decrease=min_value_increase,
+                                                   alpha=alpha)
+
+        def policy_data(tree, node_id):
+            return {'treatment': treatment_names[np.argmax(tree.value[node_id])]}
```

Comment:
I think we should also be returning the local average policy value at each node (maybe not with confidence intervals). This value is being displayed in the graphviz tree. You can use the node_dict_ that is constructed at fit time by  the interpreter. 
https://github.com/microsoft/EconML/blob/dbbd9488712f6e92566c2df588675dbe8fd0f73f/econml/cate_interpreter/_interpreters.py#L469

This is what is being used by the graphviz tree. Then you can just do: node_dict_[node_id]['mean'], node_dict_[node_id]['ci']

Reply:
Actually the node_dict just gives you the local CATE information. 
The tree.value[node_id] also contains the cost, i.e. it is the average: CATE-cost.
So maybe we should return:
'treatment': ..
'gains': ... (value - cost from tree.value[node_id])
'effect': ... (from node_dict_[node_id]['mean'])
'effect_ci': ... (from node_dict)